editoast/src/fixtures.rs

```diff
@@ -117,7 +116,7 @@ pub mod tests {
         let mut rs: RollingStockModel =
```

Comment:
```suggestion
        let mut rs: Changeset<RollingStockModel> =
```

That way you don't have to add the id in the example_*.json files.
You can use `#[model(changeset(derive(Deserialize)))]` to make the changeset deserailizable.